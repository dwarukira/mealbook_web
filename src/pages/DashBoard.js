import React, { Component } from 'react'
import { AdminNav } from "../components/NavBar"

export default class DashBoard extends Component {
  render() {
    return (
      <React.Fragment>
        <div className="">
         <AdminNav />          
        </div>
        <div className="">

        </div>
        <div className="container class">
         {this.props.children}
        </div>
      </React.Fragment>
    )
  }
}
