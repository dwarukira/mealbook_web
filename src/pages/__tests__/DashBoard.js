import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Browser } from "react-router-dom";
import DashBoard from "../DashBoard";

describe("The DashBoard Component", () => {
  it("should not regress", () => {
    const tree = renderer.create(
      <Browser>
        <DashBoard />
      </Browser>
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
