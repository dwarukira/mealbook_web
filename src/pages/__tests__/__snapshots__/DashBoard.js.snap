// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The DashBoard Component should not regress 1`] = `
Array [
  <div
    className=""
>
    <nav
        className={undefined}
    >
        <div
            className={undefined}
        >
            <a
                className={undefined}
                href="/"
                onClick={[Function]}
            >
                Menu
            </a>
        </div>
        <div
            className={undefined}
        >
            <a
                className={undefined}
                href="/d"
                onClick={[Function]}
            >
                Meals
            </a>
        </div>
        <div
            className={undefined}
        >
            <a
                className={undefined}
                href="/d/orders"
                onClick={[Function]}
            >
                Orders
            </a>
        </div>
        <div
            className={undefined}
        >
            <a
                className={undefined}
                href="/d/dashboard"
                onClick={[Function]}
            >
                Dashboard
            </a>
        </div>
    </nav>
</div>,
  <div
    className=""
/>,
  <div
    className="container class"
/>,
]
`;
